service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -359,6 +372,105 @@ public OrderWithAddressesResponseDto saveCurrentAddressForOrder(OrderAddressDtoR
         return findAllAddressesForCurrentOrder(uuid);
     }
 
+    private OrderAddressDtoRequest geoCodeSearchingRequest(String searchRequest) {
+        List<Locale> locales = List.of(new Locale("uk"), new Locale("en"));
```

Comment:
since this list is the same all the time, you can create it as
`private static final `